openadapt/db/crud.py

```diff
@@ -816,8 +817,14 @@ def acquire_db_lock(timeout: int = 60) -> bool:
             logger.error("Failed to acquire database lock.")
             return False
         if os.path.exists(DATA_DIR_PATH / "database.lock"):
-            logger.info("Database is locked. Waiting...")
-            time.sleep(1)
+            with open(DATA_DIR_PATH / "database.lock", "r") as lock_file:
```

Comment:
It appears that this path is re-used in several places. What do you think about defining it in `config.py`?